clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -599,19 +599,19 @@ def ConditionOp : CIR_Op<"condition", [
   }];
   let arguments = (ins CIR_BoolType:$condition);
   let assemblyFormat = " `(` $condition `)` attr-dict ";
+  let hasVerifier = 1;
```

Comment:
Please include documentation bits for what's the expected behavior when used within await ready regions.